addons/website/views/snippets/s_tabs.xml

```diff
@@ -3,45 +3,69 @@
 
 <template id="s_tabs" name="Tabs">
     <section class="s_tabs pt48 pb48" data-vcss="001">
-        <div class="container">
+        <div class="o_container_small">
             <div class="s_tabs_main">
-                <div class="s_tabs_nav mb-3">
-                    <ul class="nav nav-pills" role="tablist">
-                        <li class="nav-item" role="presentation">
+                <div class="s_tabs_nav mb-3" data-name="Tab Header">
+                    <ul class="nav nav-underline nav-justified flex-nowrap overflow-y-hidden overflow-x-auto" role="tablist">
+                        <li class="nav-item" role="presentation" data-name="Tab Entry">
                             <a class="nav-link active" id="nav_tabs_link_1" data-bs-toggle="tab" href="#nav_tabs_content_1" role="tab" aria-controls="nav_tabs_content_1" aria-selected="true">Home</a>
                         </li>
-                        <li class="nav-item" role="presentation">
+                        <li class="nav-item" role="presentation" data-name="Tab Entry">
                             <a class="nav-link" id="nav_tabs_link_2" data-bs-toggle="tab" href="#nav_tabs_content_2" role="tab" aria-controls="nav_tabs_content_2" aria-selected="false">Profile</a>
                         </li>
-                        <li class="nav-item" role="presentation">
+                        <li class="nav-item" role="presentation" data-name="Tab Entry">
                             <a class="nav-link" id="nav_tabs_link_3" data-bs-toggle="tab" href="#nav_tabs_content_3" role="tab" aria-controls="nav_tabs_content_3" aria-selected="false">Contact</a>
                         </li>
                     </ul>
                 </div>
                 <div class="s_tabs_content tab-content">
                     <div class="tab-pane fade show active" id="nav_tabs_content_1" role="tabpanel" aria-labelledby="nav_tabs_link_1">
                         <div class="oe_structure oe_empty">
-                            <section class="s_text_block">
-                                <div class="container s_allow_columns">
-                                    <p>Write one or two paragraphs describing your product or services.</p>
+                            <section class="s_text_image">
```

Comment:
When using (by copy/paste) a snippet this way,  remember to add `data-snippet` attribute
```suggestion
                            <section class="s_text_image" data-snippet="s_text_image">
```

Anyhow, I'm not really sure about the content used for these tabs in general.
"Tab 1" and "Tab 3" content it's the same, + the fact that the only noticeable difference it's the image flipping side on "Tab 2" is bit "misleading" 🤔.

I mean, everyone knows how tabs works, we don't have to explain it... but at first sight the content used may suggests that there is some sort of correlation/need for this content to be almost the same (?) Dunno, it's a bit weird